docker/server/entrypoint.sh

```diff
@@ -115,7 +117,7 @@ if [ -n "$(ls /docker-entrypoint-initdb.d/)" ] || [ -n "$CLICKHOUSE_DB" ]; then
     # check if clickhouse is ready to accept connections
     # will try to send ping clickhouse via http_port (max 12 retries by default, with 1 sec timeout and 1 sec delay between retries)
     tries=${CLICKHOUSE_INIT_TIMEOUT:-12}
-    while ! wget --spider -T 1 -q "http://127.0.0.1:$HTTP_PORT/ping" 2>/dev/null; do
+    while ! wget --spider -T 1 -q "http://127.0.0.1:$PORT/ping" 2>/dev/null; do
```

Comment:
This is not enough, if you are using https, you should specify https as the URL schema.

Reply:
I'll update the endpoint URL  as https://127.0.0.1:$PORT/ping.